src/main/kotlin/org/meogo/global/utill/FcmUtil.kt

```
@@ -0,0 +1,55 @@
+@Component
+class FcmUtil {
+    private val firebase: FirebaseMessaging
+        get() = FirebaseMessaging.getInstance()
+
+    fun sendMessage(fcmToken: List<String>, title: String, message: String) {
+        val message = messageSetting(fcmToken, title, message)
+
+        println("호우")
+        try {
+            firebase.sendMulticastAsync(message)
+        } catch (e: FirebaseMessagingException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)